code/modules/shuttle/manipulator.dm

```diff
@@ -159,6 +159,11 @@
 				var/mdp = action_load(S)
 				if(mdp)
 					user.forceMove(get_turf(mdp))
+					message_admins("<span class='adminnotice'>\
+						[key_name_admin(usr)] loaded [mdp] with the shuttle \
+						manipulator.</span>")
+					log_admin("<span class='adminnotice'>[key_name(usr)] \
```

Comment:
No they go straight to a .txt log; message_admins already has a span class as well.